server/api.go

```diff
@@ -665,6 +676,283 @@ func (a *API) getConnectedUsersFile(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (a *API) resetMocks(w http.ResponseWriter, r *http.Request) {
+	clientMock = nil
+	newMock := getClientMock(a.p)
+	a.p.msteamsAppClient = newMock
+	w.WriteHeader(http.StatusOK)
+}
+
+func (a *API) addMockCall(w http.ResponseWriter, r *http.Request) {
```

Comment:
Maybe a comment here explaining what this is doing would be nice. Also naming. Maybe `resetMSTeamsClientMocks` and `addMSTeamsClientMock`?

Reply:
Done